src/containers/applications/apps/assets/settings.scss

```diff
@@ -58,6 +58,8 @@
         border-radius: 4px;
         padding: 8px;
         margin: 8px 16px 20px;
+        color: white;
```

Comment:
use var(--dark-txt) instead of white.
also font-weight: 500 not required.

Reply:
Ok